test/adapters/ReduxORMAdapterSpec.js

```diff
@@ -0,0 +1,72 @@
+import '../test-helper/testUtils';
+import ReduxORMAdapter from '../../src/adapters/ReduxORMAdapter';
+import { expect } from 'chai';
+import DummyReduxORMModel from '../test-helper/DummyReduxORMModel';
+import asyncFunction from '../test-helper/asyncFunction';
+import { ORM } from 'redux-orm';
+
+const orm = new ORM();
+orm.register(DummyReduxORMModel);
+const session = orm.session();
+
+describe('ReduxORMAdapter', function () {
+  const adapter = new ReduxORMAdapter(session);
+  it('can be created', function () {
+    expect(adapter).to.be.an.instanceof(ReduxORMAdapter);
+  });
+
+  describe('#build', function () {
+    it('builds the model', asyncFunction(async function () {
+      const model = adapter.build('DummyReduxORMModel', {
+        id: 1,
+        type: 'City',
+        name: 'Vic',
+        country: 'ES',
+
+      });
+      expect(model).to.be.an.instanceof(DummyReduxORMModel);
+    }));
+  });
+
+  describe('#save', function () {
+    it('returns a promise', function () {
+      const model = adapter.build('DummyReduxORMModel', {
+        id: 1,
+        type: 'City',
+        name: 'Vic',
+        country: 'ES',
+
+      });
+      const savedModelP = adapter.save(model, DummyReduxORMModel);
+      expect(savedModelP.then).to.be.a('function');
+      return expect(savedModelP).to.be.eventually.fulfilled;
```

Comment:
There's no reason to remove the whole test - you just want to `expect(model).to.equal(savedModel)`. However you can handle that in this test:

```js
return expect(savedModel).to.eventually.equal(model)
```